airflow/contrib/operators/mysql_to_gcs.py

```diff
@@ -73,6 +75,9 @@ def __init__(self,
         :param google_cloud_storage_conn_id: Reference to a specific Google
             cloud storage hook.
         :type google_cloud_storage_conn_id: string
+        :param schema: The schema to use, if any. Should be a list of dict, or
+            json like str, i.e. [{"name"=, "type"=, "mode"=}]
```

Comment:
Maybe provide https://cloud.google.com/bigquery/docs/schemas#specifying_a_json_schema_file for reference in the doc